src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/RegexpMultiline/patchedline/expected.txt

```diff
@@ -0,0 +1,4 @@
+Starting audit...
+[ERROR] /Users/hgh/Downloads/my_file/IDEA_workspace/patchfiltergeneratepatch/src/main/java/Checker/RegexpMultiline/Test.java:18: Line matches the illegal pattern 'System\.out\.\n                print\('. [RegexpMultiline]
```

Comment:
this line is not expected as I understand, because patch changes line 19, and this violation is for number 18. This should be covered by context strategy. Am I correct?

Reply:
@baratali yes, you are correct, and I have add notes in UT about RegexpMultiline that this situation will be solved in context strategy.